### ISSUE TITLE
buffer switching gets slower with the number of open files

### ISSUE THREAD

**Issue by numerodix** [weynhamz/vim-plugin-minibufexpl#33]
With 19 buffers open switching between successive tabs takes about 1 second. With 3 buffers open it's basically instantaneous.

**Comment by numerodix** [comment]
I just realized this is fixed by disabling miniBufExplCheckDupeBufs. I can't believe the fix is so simple, this has been bugging me for months. It's even documented in the file source, but I've never read all those comments before.

**Comment by weynhamz** [comment]
Which version of MBE you are using? 'miniBufExplCheckDupeBufs' is supposing to be obsolete, it's not documented in the doc anymore in the master.

**Comment by numerodix** [comment]
It's a github checkout from I don't know exactly when, but the latest version listed at the bottom of the file is 6.4.1b5 .

**Comment by weynhamz** [comment]
It has been in 6.5 for a while, you might need to consider updating your MBE to the master.

**Comment by numerodix** [comment]
Good idea. One thing though.. In the version I was running I could cycle tabs and wrap around as well. Is that still possible?

**Comment by weynhamz** [comment]
Yes, but a little setting `g:miniBufExplCycleArround` might need to be enabled, and besides, now you can cycle the buffers in a most recently used fasion.

**Comment by numerodix** [comment]
Ah yes. The variable name looks like a misspelling, is that intentional?

**Comment by weynhamz** [comment]
Yes, they were unified to the same format a while ago, just read the doc.

发自我的 iPhone

**Comment by numerodix** [comment]
"arround" instead of "around"?

**Comment by weynhamz** [comment]
Oh,that is a misspelling,sorry. I will fix the doc soon.

发自我的 iPhone
